madlibs.php

```diff
@@ -2,49 +2,49 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <link href="https://fonts.googleapis.com/css2?family=Lexend+Mega&display=swap" rel="stylesheet">
+	<meta charset="UTF-8">
+	<link href="https://fonts.googleapis.com/css2?family=Lexend+Mega&display=swap" rel="stylesheet">
 
-    <title>Mad Libs</title>
+	<title>Mad Libs</title>
 </head>
 
 <body>
-    <form action="madlibs.php" method="post">
-        Noun:<input type="text" name="noun" /> </br>
-        Verb:<input type="text" name="verb" /> </br>
-        Adjective:<input type="text" name="adj" /> </br>
-        <input type="submit" value="submit" name="submit_btn" />
-    </form>
-    </br>
-    <?php
-    if (isset($_POST['submit_btn'])) {
-        $noun = $_POST['noun'];
-        $verb = $_POST['verb'];
-        $adj = $_POST['adj'];
-        if (!empty($_POST['noun'])) {
-            $noun = $_POST['noun'];
-        } else {
-            echo "<h3 style='color: red'>Please Choose a Noun!</h3>";
-        }
-        if (!empty($_POST['verb'])) {
-            $verb = $_POST['verb'];
-        } else {
-            echo "<h3 style='color: red'>Please Choose a Verb!</h3>";
-        }
+	<form action="madlibs.php" method="post">
+		Noun:<input type="text" name="noun" /> <br />
+		Verb:<input type="text" name="verb" /> <br />
+		Adjective:<input type="text" name="adj" /> <br />
+		<input type="submit" value="submit" name="submit_btn" />
+	</form>
+	<br />
```

Comment:
no need for the space but on line 41+ you exclude it. Consistency is king

Reply:
I don't know why it's auto-formatting to that. I just changed it and saved, but it went back to spaced.

Reply:
:shrug: the formatter is just there to "help" sometimes its up to you to fix the formatter lol

Reply:
I manually fixed all the spaces, and saved the file without formatting. Is there a better way to do this? If I command save the file, it formats back with the spaces. Maybe a better formatting extension?